packages/gatsby-theme-carbon/src/components/LeftNav/LeftNav.js

```diff
@@ -58,7 +58,10 @@ const LeftNav = props => {
         aria-label="Side navigation"
         className={classnames({
           'bx--side-nav--website': true,
-          'bx--side-nav--website--light': !props.homepage,
+          'bx--side-nav--website--dark':
```

Comment:
This might get hard to manage with multiple themes. Maybe do a switch statement on props.theme above to get the theme's class name, then just apply the class name here.

Reply:
Can you show me what you mean? was struggling to get this working in a way that made sense with a switch statement. (because of the homepage props also)

Reply:
```
  const getClassName = () => {
    switch (props.theme) {
      case 'gray100':
        return 'bx--side-nav--website--dark';
      default:
        return 'bx--side-nav--website--light';
    }
  };
```

I can add that, but not sure how best to handle homepage